dashboard/lib/services/lesson_import_helper.rb

```diff
@@ -6,6 +6,25 @@ def self.update_lockable_lesson(script_levels, lesson_id)
     [create_activity_with_levels(script_levels, lesson_id, 1)]
   end
 
+  def self.create_lesson_resources(cb_resources, course_version_id)
+    cb_resources.map do |cb_resource|
+      raise unless cb_resource['slug']
+      resource = Resource.find_or_initialize_by(
+        course_version_id: course_version_id,
+        key: cb_resource['slug']
+      )
+      resource.name = cb_resource['name']
+      resource.url = cb_resource['url']
+      resource.type = cb_resource['type']
+      resource.audience = cb_resource['student'] ? 'Student' : 'Teacher'
```

Comment:
I am assuming that there is never a case where we would want to set this to Verified Teacher during an import, because data on CB is public. Is there a future step to try to pull these out of the Student Facing Lesson Plan levels in each lesson?

Reply:
What do you mean by "these"? Do you mean teacher resouces?

I'm still not familiar with student facing lesson plans so I don't know the answer here. Definitely agree that none of the imported resources should have the audience "Verified Teacher" though.

Reply:
I guess I am naively asking if we should be creating any new Resource objects with audience "Verified Teacher" (which I referred to as "these"), and if so, what the data source would be for those. I don't think there will be time to do this before Friday, but want to make sure we are tracking any future work.

My first guess is that we might want to create new Verified Teacher Resources from what you see in the "For Teachers Only" box on "Lesson Overview" pages a.k.a. "Student-Facing Lesson Plans". @tess323 , do you have a sense of whether this is the end goal? If so, do we know if it is tied to January or April deadlines? or possibly bundled with to-be-scheduled work on Student-Facing Lesson Plan? Here is an example: https://levelbuilder-studio.code.org/s/csd3-2020/stage/1/puzzle/1
![Screen Shot 2020-11-18 at 10 39 37 AM](https://user-images.githubusercontent.com/8001765/99573027-6617e600-298a-11eb-9740-40b6b3c0d72b.png)

@bethanyaconnor , you may be correct that we (instead? also?) want to create new `Resource` objects for existing Teacher Resources in the menu on the script overview page:
![Screen Shot 2020-11-18 at 10 42 08 AM](https://user-images.githubusercontent.com/8001765/99573279-be4ee800-298a-11eb-8d0e-f5ccf02ab0f8.png)

We have a line item "Teacher Resources on Unit and Curriculum use New Resource Model" on our Roadmap for January launch, so I think we are good in terms of tracking this. One thing to note is that at least some of these Teacher Resources appear to be visible to non-verified teachers, so those would probably become Resource objects with audience "Teacher", not "Verified Teacher".

Reply:
ahhh thanks for the explanation!

Reply:
per discussion in stand-up, "Verified Teacher" resources are a new feature, so there is nothing to import here for January or April.